### DISCUSSION TITLE
Twitter.com is dead and now the only way to view it is through x.com. Can the extension be updated to work there as well?

### DISCUSSION THREAD

**Discussion by amethyst-deceiver** [kheina-com/Blue-Blocker#284] [Ideas]
Unless I'm missing something, it's pretty much useless now.

**Comment by NuraxGH** [comment]
Same for me, they need to update it for the new IP

**Comment by kheina** [comment]
https://github.com/kheina-com/Blue-Blocker/releases/tag/v0.4.2
